react/components/ProductSummaryImage/ImagePlaceholder.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+
```

Comment:
This PR is based on [this branch](https://github.com/vtex-apps/product-summary/pull/194), so there are some commits here related to that PR